src/renderer/components/Bonds/table/BondsTable.styles.ts

```diff
@@ -3,6 +3,7 @@ import { Col as ACol } from 'antd'
 import styled from 'styled-components'
 import { palette } from 'styled-theme'
 
+import { ConfirmationModal as ConfirmationModalUI } from '../../modal/confirmation'
```

Comment:
[nitpick] `ConfirmationModalUI` -> `UIConfirmationModal` 

Just to keep name convention we already have at other places (`UIButton`,, `UILabel` etc.)